src/tools/perf/perftest_params.c

```diff
@@ -87,8 +87,8 @@ static void usage(const struct perftest_context *ctx, const char *program)
     printf("     -o             do not progress the responder in one-sided tests\n");
     printf("     -B             register memory with NONBLOCK flag\n");
     printf("     -b <file>      read and execute tests from a batch file: every line in the\n");
-    printf("                    file is a test to run, first word is test name, the rest of\n");
-    printf("                    the line is command-line arguments for the test.\n");
+    printf("                    file is a test to run, the first word is a user-defined test name,\n");
+    printf("                    the rest of the line is command-line arguments for the test.\n");
```

Comment:
This part ("the rest of..") is a bit vague.
It is unclear that the user still has to specify `-t ...` in each line (the "test name" doesn't replace it).
Maybe either remove the "for the test" suffix, or make it more specific and clearer?

Reply:
How about "read and execute tests from a batch file: every line in the file is a test to run, the first element is a user-defined test name, followed by '-t <test>' option to specify the test type and command-line arguments for the test"? @gleon99

Reply:
Maybe:
```
read and execute tests from a batch file: every line in the file is a test to run, the first element is a user-defined test name, followed by '-t ' option and command-line arguments
```
to make it shorter.

Reply:
fixed @gleon99